config.gradle

```diff
@@ -175,34 +175,17 @@ if (!project.hasProperty('onlyAthena')) {
                 compilerFamily = 'Clang'
                 detectPlatform = mac64PlatformDetect
             }
-            raspbian(CrossBuildConfig) {
```

Comment:
Does this not work? Why no build for the PI anymore?

Reply:
It was running into both maintainability issues and other extraction issues that were difficult to solve. The solution for now is to make all of them build as a `nativearm` classifier. It makes the custom user case easier, and the custom cross case easier as well. And it was never really tested and kind of a pain to work with as well.